File: src/test/java/com/matag/language/StringUtilsTest.java
package com.matag.language;

import static com.matag.language.StringUtils.replaceLast;
import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

public class StringUtilsTest {
  @Test
  public void replaceLastTest() {
    assertThat(replaceLast("cat dog cat", "cat", "other")).isEqualTo("cat dog other");
  }
}

File: src/test/kotlin/com/matag/language/StringUtilsTest.kt
package com.matag.language

import com.matag.language.StringUtils.replaceLast
import org.assertj.core.api.Assertions
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class StringUtilsTest {
    @Test
    fun replaceLastTest() {
        assertThat(replaceLast("cat dog cat", "cat", "other")).isEqualTo("cat dog other")
    }
}